src/cashier/withdraw.es6

```diff
@@ -7,12 +7,11 @@ import liveapi from 'websockets/binary_websockets';
 import windows from 'windows/windows';
 import rv from 'common/rivetsExtra';
 import currencyDialog from 'cashier/currency';
-import _ from 'lodash';
+import {debounce} from 'lodash'
```

Comment:
@arnabk I tried that and it doesn't seem to work because on `import debounce from 'lodash/debounce'` it actually sends a request to **https://localhost/<version>/lib/lodash/dist/lodash.min/debounce.js** which doesn't exist. We might have to change something in the config to make it work.